experiments/cbow.py

```diff
@@ -0,0 +1,171 @@
+from typing import Dict, List, Any
+import logging
+
+import numpy as np
+import pandas as pd
+import torch
+
+from transfer_nlp.common.tokenizers import CustomTokenizer
+from transfer_nlp.embeddings.embeddings import Embedding
+from transfer_nlp.loaders.loaders import DatasetSplits, DataFrameDataset, DatasetHyperParams
+from transfer_nlp.loaders.vectorizers import VectorizerNew
+from transfer_nlp.loaders.vocabulary import CBOWVocabulary
+from transfer_nlp.plugins.config import register_plugin
+from transfer_nlp.plugins.helpers import ObjectHyperParams
+from transfer_nlp.plugins.predictors import Predictor, PredictorHyperParams
+
+
+name = 'transfer_nlp.experiments.cbow'
+logging.getLogger(name).setLevel(level=logging.INFO)
+logger = logging.getLogger(name)
+
+# Vectorizer
+@register_plugin
+class CBOWVectorizer(VectorizerNew):
```

Comment:
`VectorizerNew`? maybe we should rename this before merging

Reply:
I changed to `Vectorizer`